aodndata/moorings/products_handler.py

```diff
@@ -111,6 +112,7 @@ def __init__(self, *args, **kwargs):
         self.input_file_collection = None
         self.input_file_variables = None
         self.excluded_files = dict()
+        self.product_qc_flags = [[1, 2], [0, 1, 2]]
 
```

Comment:
maybe a comment about why the nested lists or using a dictionary?

Reply:
This is only relevant to the hourly product. I've removed the class attribute, hard-coded the values in the `_make_hourly_timeseries` method, and added a comment to explain it.

Reply:
I understand, but to know what this will cause down the road was a bit convoluted since the aodntools function that uses it got several parameters, several defaults, and is 500lines or more. 
For example, if you use the actual names inside the aodntools function (hourly-non-qc-whatver) and do string comparison instead you make a safe-closure. If someone add another case here (e.g. [0,1]) the [0,1,2] collection is overwritten.

Reply:
Good point, but fixing that will require an API change to the function in aodntools. I've added this as a future enhancement (https://github.com/aodn/python-aodntools/issues/100)